package.json

```diff
@@ -13,6 +13,7 @@
     "@emotion/server": "^11.4.0",
     "@emotion/styled": "^11.8.1",
     "@fontsource/roboto": "^4.5.3",
+    "@material-ui/core": "^4.12.3",
```

Comment:
this package is now depreciated
https://www.npmjs.com/package/@material-ui/core